src/public/class-sesamy-content-container.php

```diff
@@ -175,10 +175,25 @@ public function show_paywall( $post, $post_settings ) {
 
 		ob_start();
 
+		$post_settings = sesamy_get_post_settings( $post_id );
+		if ( $post_settings['access_level'] != -1 && !empty( $post_settings['access_level'] ) ) {
+			$access_level = $post_settings['access_level'];	
+		} else {
+			$access_level = 'entitlement';
+		}
+
+		$html_attributes_l = array(
+			"access-level" => $access_level,
+		);
+	
+		echo '<sesamy-locked-content-container ';
+			Sesamy_Utils::html_attributes( $html_attributes_l );
```

Comment:
I think we're missing some attributes here, the `<sesamy-locked-content-container>` should also include:
- `required-attributes`: for the case that the access-level is logged-in, we need to know if there are some attributes required as well.
- `publisher-content-id`: to get the item-src from the content data in the case that the item-src is not set as an attribute.
- `item-src`: for the case that the access-level is entitlement, we need to know which entitlement should be in charge of locking/unlocking the content.
- `pass`: idem item-src.

These valules should be the same the we pass to the `<sesamy-content-container>` component.